app/javascript/components/explore/ExploreDisplayPanelManager.jsx

```diff
@@ -477,6 +469,21 @@ export default function ExploreDisplayPanelManager({
                     </div>
                   </>
                 }
+                { showCellFiltering && !allowCellFiltering &&
+                  <>
+                    <div className="row">
+                      <div className={`col-xs-12 cell-filtering-button ${shownTab === 'scatter' && !studyHasDe ? 'create-annotation-cell-filtering' : ''}`}>
+                        <button
+                          disabled="disabled"
+                          className={`btn btn-primary`}
```

Comment:
For analytics on whether users click this, which would signal interest in cell filtering in the disabled context:

```suggestion
                          className={`btn btn-primary cell-filtering-button-disabled`}
```